pipelines/deployer/deployer.tf

```diff
@@ -85,6 +85,15 @@ variable "enable_nlb" {
   default = "0"
 }
 
+variable "cls_destination_enabled" {
+  type    = string
```

Comment:
can this be a numeric type instead? then we can just say

```terraform
count           = var.cls_destination_enabled 
```

instead of the ternary operator stuff?

Reply:
That wouldn't match how enable_nlb or splunk_enabled work, we probably should have a ternary for these to ensure count can never be set to something other than 0 or 1.